meinberlin/apps/budgeting/assets/BudgetingProposalList.jsx

```diff
@@ -48,6 +50,21 @@ export const BudgetingProposalList = (props) => {
               pageCount={meta.page_count}
               onPaginate={newUrl => onPaginate(newUrl)}
             />}
+        </>
+      )
+    } else {
+      list = (
+        <span>{django.gettext('Nothing to show')}</span>
```

Comment:
Did you add this new empty state only for the test or was that something we wanted to have anyway?

Reply:
this was there in the django template but was forgotton in the jsx, adding test meant that I found this out (even if test isn't quite working)